appyx-components/internal/common/src/commonMain/kotlin/com/bumble/appyx/components/internal/testdrive/ui/TargetUiState.kt

```diff
@@ -2,11 +2,13 @@ package com.bumble.appyx.components.internal.testdrive.ui
 
 import com.bumble.appyx.interactions.core.ui.property.impl.BackgroundColor
 import com.bumble.appyx.interactions.core.ui.property.impl.Position
+import com.bumble.appyx.interactions.core.ui.property.impl.RotationZ
 import com.bumble.appyx.interactions.core.ui.state.MutableUiStateSpecs
 
 @Suppress("unused")
 @MutableUiStateSpecs
 class TargetUiState(
     val position: Position.Target,
+    val rotationZ: RotationZ.Target,
```

Comment:
If we add the rotation to the default ui, it will probably mess up the screenshot tests. We can make this the new baseline, or we can make it a separate example.